docs/release/kubernetes-upgrade.md

```diff
@@ -259,10 +293,7 @@ A later version can point to those arguments with no change:
     serverArgs: *serverArgs-v1
 ```
 If you are unsure of the new minor versions min/max constraints you can ask the Project manager and/or QA.
-# Create system-agent-installer-k3s Release Images
```

Comment:
I think this is useful but maybe rephrased since it's not a manual step but something to check on and verify the tag was created.